src/Shared/Im/View/ChatInput.purs

```diff
@@ -127,9 +127,9 @@ chatBarInput
                                     <>
                                           [ HA.class' "chat-input"
                                           , HA.id $ show elementId
-                                          , HA.placeholder $ if isWebSocketConnected then "Type here to message " <> recipientName else "Waiting for connection..."
-                                          , HA.disabled $ not isWebSocketConnected
-                                          , SK.keyDownOn "Enter" EnterBeforeSendMessage
+                                          , HA.placeholder $ if isWebSocketConnected then "Type here to message " <> recipientName else "Can't send message while no connection"
+                                          --, HA.disabled $ not isWebSocketConnected -- change this to disable sending
+                                          , SK.keyDownOn "Enter" $ if isWebSocketConnected then EnterBeforeSendMessage else ResizeChatInput
```

Comment:
This is an in between solution, just to try to see if I can disable sending with enter. 
This makes enter work the same way as it does when `Send message on enter` is disabled, strictly when there is no connection. 
Ideally, it should do nothing, but I didn't find a function that works that way just yet.

Reply:
currently keyDownOn has type  Key → (Event → ImMessage) → NodeData ImMessage

you can adapt it to Key → (Event → Maybe ImMessage) → NodeData ImMessage